apps/web/src/app/events/components/EventDescription.tsx

```diff
@@ -1,5 +1,20 @@
-import { Text } from "@dotkomonline/ui"
+import { ReadMore, Text } from "@dotkomonline/ui"
 
-export function EventDescription({ description }: { description: string }) {
-  return <Text className="whitespace-pre-line">{description}</Text>
+export function EventDescription({ description }: { description: string | null }) {
+  if (!description) {
```

Comment:
This should probably be `description?: string` as passing `null` explicitly as a prop is awkward